BraintreeCore/src/main/java/com/braintreepayments/api/core/AnalyticsClient.kt

```diff
@@ -161,7 +171,12 @@ internal class AnalyticsClient(
         )
         val event = AnalyticsEvent(name = "crash", timestamp = timestamp)
         val eventJSON = mapAnalyticsEventToFPTIEventJSON(event)
-        val eventBlobs = listOf(AnalyticsEventBlob(eventJSON))
+        val eventBlobs = listOf(
+            AnalyticsEventBlob(
+                jsonString = eventJSON,
+                sessionId = ""
```

Comment:
The reportCrash function doesn't end up using the `sessionId` in `AnalyticsEventBlob` as it uses the `sessionId` in the deviceMetadata. But unfortunately `AnalyticsEventBlob.sessionId` is a required field.

I think the more correct approach here would be to remove the use of `AnalyticsEventBlob` when reporting a crash since we're not interacting with the DB. We could create a separate data model for the FPTI request. But I didn't want to introduce to big of a change leading up to the release of v5, but open to other thoughts!